GameData/ROEngines/RealPlume/RD270M_CH4_RealPlume.cfg

```diff
@@ -0,0 +1,31 @@
+@PART[ROE-RD270M]:FOR[RealPlume]:NEEDS[SmokeScreen,!Waterfall]
+{
+	@MODULE[ModuleEngines*]
+	{
+		%powerEffectName = Pentaborane_Lower
+	}
+	@MODULE[ModuleEngineConfigs]
+	{
+		%type = ModuleEnginesRF
+	}
```

Comment:
Delete.  Do not change MEC type in a Plume patch.  (Let RO global patcher fix this up, that's part of its job.)